Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM alpine:3.12
 
 ARG KUBE_VERSION="v1.16.10"
+ENV AWS_DEFAULT_REGION us-west-1
+ENV AWSCLI_VERSION 1.14.50
```

Comment:
not really blocking, but...

awscli v1.14.50 is 3 years old, any reason not to use a more recent version?

Reply:
bumping to 1.19.107